pypsa/optimization/expressions.py

```diff
@@ -533,7 +534,7 @@ def operation(
         aggregate_groups: str = "sum",
         aggregate_across_components: bool = False,
         at_port: Sequence[str] | str | bool = False,
-        groupby: Callable | None = None,
+        groupby: list[str] | Callable | None = None,
```

Comment:
Wouldn't a nicer API be `str | list[str] | Callable`?

Reply:
yes, that was already supported - I added the type annotation